README.md

```diff
@@ -1 +1,71 @@
-# airflow-provider-flyte
+# Flyte Provider for Apache Airflow
+
+This package provides an operator, a sensor, and a hook that integrates [Flyte](flyte.org/) into Apache Airflow.
+`FlyteOperator` is helpful to trigger a task/workflow in Flyte and `FlyteSensor` enables monitoring a Flyte execution status
+for completion.
+
+## Installation
+
+Prerequisites: An environment running `apache-airflow`.
+
+```
+pip install airflow-provider-flyte
+```
+
+## Configuration
+
+In the Airflow UI, configure a _Connection_ for Flyte.
+
+- Host(optional): The FlyteAdmin host. Defaults to localhost.
+- Port (optional): The FlyteAdmin port. Defaults to 30081.
+- Login (optional): `client_id`
+- Password (optional): `client_credentials_secret`
+- Extra (optional): Specify the `extra` parameter as JSON dictionary to provide additional parameters.
+  - `project`: The default project to connect to.
+  - `domain`: The default domain to connect to.
+  - `insecure`: Whether to use SSL or not.
+  - `command`: The command to execute to return a token using an external process.
+  - `scopes`: List of scopes to request.
```

Comment:
ideally you do not need this. But if you need for certain cases, then you should support all of these - https://github.com/flyteorg/flytekit/blob/195d26ab0cc03260bb96b828219fcf7e6ea768a1/flytekit/configuration/internal.py#L61

Reply:
I've already added the `PlatformConfig`-related params. Let me also add the other config params: https://github.com/flyteorg/flytekit/blob/195d26ab0cc03260bb96b828219fcf7e6ea768a1/flytekit/configuration/__init__.py#L483.